core-developers/experts.rst

```diff
@@ -56,7 +56,7 @@ __future__
 __main__              gvanrossum, ncoghlan
 _thread
 abc
-argparse
+argparse              savannahostrowski*
```

Comment:
Should Serhiy be here too? (@serhiy-storchaka)
```suggestion
argparse              savannahostrowski*, serhiy-storchaka
```

Reply:
I'll leave that up to him! :)